odoo/tools/config.py

```diff
@@ -813,6 +815,11 @@ def _load_file_options(self, rcfile):
         try:
             p.read([rcfile])
             for (name,value) in p.items('options'):
+                if name == 'without_demo':
```

Comment:
that we need a special case is ugly as fuck!

Reply:
It is fine, since we are replacing this option and it will become deprecated.